sql/api/src/main/scala/org/apache/spark/sql/errors/QueryParsingErrors.scala

```diff
@@ -387,6 +387,13 @@ private[sql] object QueryParsingErrors extends DataTypeErrorsBase {
       ctx)
   }
 
+  def invalidStatementError(operation: String, ctx: ParserRuleContext): Throwable = {
+    new ParseException(
+      errorClass = "INVALID_STATEMENT_OR_CLAUSE",
+      messageParameters = Map("operation" -> operation),
```

Comment:
How about to quote/wrap it by `toSQLStmt()`.

Reply:
Done. I cannot yet remove the temp because there are more cases where the message should be dispatched to an unsupported feature with subclass. May do these in a separate PR.